site/static/docs/5.2/assets/brand/orange-logo-formatted.svg

```diff
@@ -0,0 +1,28 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 283.46 283.46">
```

Comment:
Do we keep such a weird viewbox ?

Reply:
It was designed like this so we don't have a choice.